packages/base/src/types.ts

```diff
@@ -2,6 +2,8 @@ import { IDict, IJupyterGISWidget } from '@jupytergis/schema';
 import { WidgetTracker } from '@jupyterlab/apputils';
 import { Map } from 'ol';
 
+import { COLOR_RAMP_DEFINITIONS } from '@/src/dialogs/symbology/colorRampUtils';
```

Comment:
This creates a circular dependency between this module and `colorRampUtils`. Let's extract `COLOR_RAMP_DEFINITIONS` to `dialogs/symbology/constants` or something along those lines.